cmd/proxy/main.go

```diff
@@ -464,9 +464,13 @@ func createVersionsRegistry(
 		return nil, err
 	}
 
-	newTxCostProcessor := func() process.TransactionCostHandler {
-		txCostProc, _ := txcost.NewTransactionCostProcessor(bp, pubKeyConverter)
-		return txCostProc
+	newTxCostProcessor := func() (process.TransactionCostHandler, error) {
+		return txcost.NewTransactionCostProcessor(
+			bp,
+			pubKeyConverter,
+			ecConf.FeeSettings.MaxGasLimitPerBlock,
```

Comment:
please make sure the `economics.toml` file is up-to-date with `elrond-go`

Reply:
updated the economics.toml file